<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu_Utama">
    <property name="title">
     <string>Menu Utama</string>
    </property>
    <addaction name="actionData_Admin"/>
    <addaction name="actionData_Merek"/>
    <addaction name="actionData_Mobil"/>
    <addaction name="actionData_Member"/>
    <addaction name="actionData_Tawar"/>
   </widget>
   <addaction name="menuMenu_Utama"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionData_Admin">
   <property name="text">
    <string>Data Admin</string>
   </property>
  </action>
  <action name="actionData_Merek">
   <property name="text">
    <string>Data Merek</string>
   </property>
  </action>
  <action name="actionData_Mobil">
   <property name="text">
    <string>Data Mobil</string>
   </property>
  </action>
  <action name="actionData_Member">
   <property name="text">
    <string>Data Member</string>
   </property>
  </action>
  <action name="actionData_Tawar">
   <property name="text">
    <string>Data Tawar</string>
   </property>
  </action>
  <action name="actionData_Barang">
   <property name="text">
    <string>Data Barang</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
